core/src/main/java/org/secondbase/core/config/SecondBaseModule.java

```diff
@@ -0,0 +1,14 @@
+package org.secondbase.core.config;
+
+import org.secondbase.core.SecondBase;
+
+/**
+ * Implemented by SecondBase modules. Enables loading of Flags and setting active SecondBase
```

Comment:
This is the inteface that "needs to be implemented," I think that would sound better.